@xen-orchestra/fs/src/s3.js

```diff
@@ -16,16 +17,20 @@ export default class S3Handler extends RemoteHandlerAbstract {
   constructor(remote, _opts) {
     super(remote)
     const { host, path, username, password } = parse(remote.url)
-
     // https://www.zenko.io/blog/first-things-first-getting-started-scality-s3-server/
-    this._s3 = new AWS.S3({
+    const options = {
```

Comment:
Why have you moved options out of the construction?

Please try to keep changes to a minimum :slightly_smiling_face: